desk/app/profile.hoon

```diff
@@ -134,6 +137,12 @@
     ::
     :-  [%pass /eyre/cache %arvo %e %set-response '/profile' `[| %payload payload]]
     (spout:rudder id payload)
+  ?:  =('/expose/style/shared.css' url.request)
+      :-  [200 ['content-type' 'text/css']~]
+      `(as-octs:mimes:html style-shared)
+  ?:  =('/profile/page.css' url.request)
+      :-  [200 ['content-type' 'text/css']~]
+      `(as-octs:mimes:html style-page)
```

Comment:
You might consider serving this at `/profile/style/page.css` instead, for consistency with the `/expose/style/*.css` paths. Or maybe we should take out the `/style` in both?

(It doesn't strictly need to match the file/folder structure in the desk, but I'd update those to match if you make such changes.)

Reply:
Oh and we'll want to make sure this gets served from cache at some point, but, fine for now.

Reply:
Moved!